cmd/snap/cmd_debug_state.go

```diff
@@ -227,6 +234,201 @@ func (c *cmdDebugState) showIsSeeded(st *state.State) error {
 	return nil
 }
 
+// connState is the connection state stored by InterfaceManager
+type connState struct {
```

Comment:
Can you add a note to the corresponding struct on the other side that those should be synchronized please.

Reply:
Added.